packages/good-design/src/stories/apps/goodid/GoodId.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { GoodIdCard } from "../../../apps/goodid";
+import { W3Wrapper } from "../../W3Wrapper";
+
+const mockCredential = [
+  {
+    credentialType: "VerifiableLocationCredential",
+    verifiedValue: "US"
+  },
+  {
+    credentialType: "VerifiableAgeCredential",
+    verifiedValue: 24
+  }
+];
+
+//todo: add expiration date utilty example
+export const GoodIdCardExample = () => (
+  <GoodIdCard
+    credentialsList={mockCredential}
+    account="0x000...0000"
+    isWhitelisted={false}
+    fullname="Just a name"
+    expiryDate="Expires on April 12, 2023"
```

Comment:
expiry should be from the credential

Reply:
the hook to retrieve expiry date of the fv-id is part of this PR: https://github.com/GoodDollar/GoodWeb3-Mono/pull/130

Reply:
every certificate has expiration date

Reply:
So the flow is:
1. if certificate exists > take expiry date from certificate
2. if whitelisted but no certificate > verify fv expiration through hook > pass down to goodid card

Right?

Reply:
lets leave it like this at the moment. if you already have that working.